parsec/backend/pki.py

```diff
@@ -138,6 +142,9 @@ async def api_pki_enrollment_submit(
                 submit_payload_signature=msg["submit_payload_signature"],
                 submit_payload=msg["submit_payload"],
                 submitted_on=submitted_on,
+                submitter_der_x509_certificate_email=msg.get(
```

Comment:
msg contains the default value even if the sent payload was missing it (it's the deserialization's job)

![image](https://user-images.githubusercontent.com/3187637/168564048-098171bf-6576-4578-a6ef-3244d5b7c857.png)